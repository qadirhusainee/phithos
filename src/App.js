import "react-native-gesture-handler";
import { enableES5 } from "immer";
import React from "react";
import type { FunctionComponent } from "react";
import { enableScreens } from "react-native-screens";
import { Provider } from "react-redux";
import { PersistGate } from "redux-persist/integration/react";

import Route from "./Route";
import configureStore from "./redux/configureStore";

enableES5();

const { store, persistor } = configureStore();

const App: FunctionComponent = () => {
  return (
    <Provider store={store}>
      <PersistGate loading={null} persistor={persistor}>
        <Route />
      </PersistGate>
    </Provider>
  );
};

export default App;
