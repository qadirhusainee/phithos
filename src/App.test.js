import React from "react";
import App from "./App";

import renderer from "react-test-renderer";

describe("App", () => {
  it("renders correctly", () => {
    renderer.create(<App />);
  });

  it("Take snapshot", async () => {
    const tree = await renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
