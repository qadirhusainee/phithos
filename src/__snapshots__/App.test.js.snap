// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Take snapshot 1`] = `null`;
